libraries/AP_OSD/AP_OSD_MSP_DisplayPort.cpp

```diff
@@ -141,4 +189,276 @@ float AP_OSD_MSP_DisplayPort::get_aspect_ratio_correction() const
 }
 
 
+#if AP_MSP_INAV_FONTS_ENABLED
+/*
+  the first column in the code ID, 2nd column is the code page
+ */
+const uint8_t AP_OSD_MSP_DisplayPort::ap_to_inav_symbols_map[256][2] {
+        {0x00,0},
```

Comment:
These are defined as symbols in AP_OSD_Backend.h. I think you should use the symbols rather than the numbers where possible here otherwise its going to be very difficult to spot errors / make changes.

Reply:
comments added...it was worthwhile because I found that the batt symbol fonts had not been completely mapped, only the start of the sequence...unlike other symbol sequences the length of sequence is hard coded rather than a value in the symbol  define table